### PR TITLE
Update compiler plugin code action text edit range

### LINKED ISSUE #34581: Applying a compiler plugin code action automatically duplicates commented out code
**Description:**

Refer to the screen recording below where executing the `Replace with '2.0'` code action automatically inserts a duplicated code segment that was commented out.

https://user-images.githubusercontent.com/27485094/148741278-4df2272a-e358-4dab-8d78-5379b74b9ca3.mov

**Steps to reproduce:**

```
// import ballerina/io;

// // Anonymous function syntax.
// var isOdd = function(int n) returns boolean {
//     return n % 2 != 0;
// };

// // Function type syntax.
// type IntFilter function (int n) returns boolean;


public function main() {
    float x = 2.;
}

```

**Affected Versions:**
SL Beta6

### PR DESCRIPTION
**Pull request by IMS94**
## Purpose
$subject

Fixes #34581

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by nadeeshaan — DISMISSED** [review]
LGTM

File: tests/language-server-integration-tests/src/test/resources/command/compiler-plugins/source/source1.bal [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+   
+   // Var assignment is required below
+   createInt();
+}
+
+function createInt() returns int {
+    return 1;
+}
+
+// A traling comment here
```
Comment (TharushiJay) [inline tests/language-server-integration-tests/src/test/resources/command/compiler-plugins/source/source1.bal:15]: Small typo and missing newline
```suggestion
// A trailing comment here

```

**Review by malinthar — DISMISSED** [review]
LGTM

File: tests/language-server-integration-tests/src/test/resources/command/compiler-plugins/source/source1.bal (hunk 1 above)
Comment (IMS94) [reply tests/language-server-integration-tests/src/test/resources/command/compiler-plugins/source/source1.bal]: Done!

**Review by mohanvive — APPROVED** [review]

**Pull request merged**